ironmon_tracker/data/DataHelper.lua

```diff
@@ -174,15 +174,15 @@ function DataHelper.buildTrackerScreenDisplay(forceView)
 
 	-- Update: Held Item and Ability area(s)
 	data.p.line1 = Constants.BLANKLINE
-	data.p.line2 = Constants.BLANKLINE
-	if data.x.viewingOwn then
+    data.p.line2 = Constants.BLANKLINE
+    if data.x.viewingOwn then
 		if viewedPokemon.heldItem ~= nil and viewedPokemon.heldItem ~= 0 then
 			data.p.line1 = MiscData.Items[viewedPokemon.heldItem]
 		end
 		local abilityId = PokemonData.getAbilityId(viewedPokemon.pokemonID, viewedPokemon.abilityNum)
-		if abilityId ~= nil and abilityId ~= 0 then
-			data.p.line2 = AbilityData.Abilities[abilityId].name
-		end
+        if abilityId ~= nil and abilityId ~= 0 then
+            data.p.line2 = AbilityData.Abilities[abilityId].name
+        end
```

Comment:
This is super minor and nitpicky, as it's more a consistency thing lol but our `.editorconfig` file specifies tabs instead of spaces (if you use VSCode there's a EditorConfig extension that applies these config settings)